test/e2e/swaps/shared.js

```diff
@@ -63,8 +62,137 @@ const buildQuote = async (driver, options) => {
   await driver.clickElement('.searchable-item-list__primary-label');
 };
 
+const reviewQuote = async (driver, amount, swapFrom, swapTo) => {
```

Comment:
nit: for consistency, can we have two params for this and other shared functions? First one would be `driver` and second one an object with options, so it would look like this:
```
const reviewQuote = async (driver, { amount, swapFrom, swapTo }) => {
```

Reply:
Good suggestion. I have changed the code